cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -234,6 +234,8 @@ public static final class AppFabric {
       = "app.program.local.dataset.deleter.initial.delay";
     public static final String PROGRAM_TERMINATOR_INTERVAL_SECS = "app.program.terminator.interval.secs";
     public static final String PROGRAM_TERMINATE_TIME_BUFFER_SECS = "app.program.terminate.time.buffer.secs";
+    public static final String TETHERED_PROGRAM_TERMINATE_TIME_SECS
+      = "app.program.tethered.terminate.time.buffer.secs";
```

Comment:
nit: think it's slightly better to have the same prefix - app.program.terminate.tethered.time.buffer.secs.

that way if somebody sorts on key, they'll be next to each other.

Reply:
fixed